src/quickstart/smart-contract.md

```diff
@@ -235,56 +269,51 @@ In order to deploy a contract, you need to have a wallet to sign the transaction
 
 ```console
-Contract id: 0xd09b469b0c31c05222b553021aa23c3b6a535db5092c22b84690dc88ca17deaa
+forc deploy --testnet
 ```
 
-Be sure to save this as you will need it to build a frontend with the Typescript SDK later in this tutorial.
+The terminal will ask for the password of the wallet:
 
-The terminal will output a `Transaction id to sign` and prompt you for a signature. Open a new terminal tab and view your accounts by running `forc wallet accounts`. If you followed these steps, you'll notice you only have one account, `0`.
+`Please provide the password of your encrypted wallet vault at "~/.fuel/wallets/.wallet":`
 
-Grab the `Transaction id to sign` from your other terminal and sign with your account by running the following command:
+Once you unlocked the wallet, the terminal will show a list of the accounts of this wallet:
 
 ```console
-forc wallet sign --account `[the account index, without brackets]` tx-id `[Transaction id to sign, without brackets]`
+Account 0 -- fuel18caanqmumttfnm8qp0eq7u9yluydxtqmzuaqtzdjlsww5t2jmg9skutn8n:
+  Asset ID                                                           Amount
+  0000000000000000000000000000000000000000000000000000000000000000 499999940
 ```
 
-Your command should look like this:
+Just below the list, you'll see this prompt:
 
-```console
-$ forc wallet sign --account 0 tx-id 16d7a8f9d15cfba1bd000d3f99cd4077dfa1fce2a6de83887afc3f739d6c84df
-Please enter your password to decrypt initialized wallet's phrases:
-Signature: 736dec3e92711da9f52bed7ad4e51e3ec1c9390f4b05caf10743229295ffd5c1c08a4ca477afa85909173af3feeda7c607af5109ef6eb72b6b40b3484db2332c
-```
+`Please provide the index of account to use for signing:`
 
-Enter your password when prompted, and you'll get back a `signature`. Save that signature, and return to your other terminal window, and paste that in where its prompting you to `provide a signature for this transaction`.
+Then you'll enter the number of the account of preference and press `Y` when prompted to accept the transaction.
 
-Finally, you will get back a `TransactionId` to confirm your contract was deployed. With this ID, you can head to the [block explorer](https://fuellabs.github.io/block-explorer-v2/) and view your contract.
+Finally, you will get back the network endpoint where the contract was dpeloyed, a `ContractId` and the block where the transaction was signed. To confirm your contract was deployed. With this ID, you can head to the [block explorer](https://fuellabs.github.io/block-explorer-v2/) and view your contract.
 
-> **Note**
-> You should prefix your `TransactionId` with `0x` to view it in the block explorer
+```console
+Contract deploy-to-beta-4 Deployed!
+
+Network: https://beta-4.fuel.network
+Contract ID: 0x8342d413de2a678245d9ee39f020795800c7e6a4ac5ff7daae275f533dc05e08
+Deployed in block 0x4ea52b6652836c499e44b7e42f7c22d1ed1f03cf90a1d94cd0113b9023dfa636
+```
 
 ![block explorer](../images/block-explorer.png)
 
```

Comment:
Since we link to it, we will also need to update the code here: https://github.com/FuelLabs/quickstart